### ISSUE TITLE
Chp 7 - Ch7_DistDQN.ipynb

### ISSUE THREAD

**Issue by leecming** [DeepReinforcementLearning/DeepReinforcementLearningInAction#9]
Is there an error in the training loop code for playing Atari-Freeway: specifically generating the predictions?

_pred2_batch = dist_dqn(state2_batch.detach(),theta_2,aspace=aspace)_

Should the state2_batch be state_batch?

**Comment by outlace** [comment]
I believe the original is correct. Does it not work for you?
